app/scripts/modules/core/src/search/infrastructure/infrastructure.less

```diff
@@ -105,10 +105,6 @@
       background-color: var(--color-alabaster);
       height: 200px;
       overflow-y: auto;
-      .search-result {
-        display: inline-block;
-        width: calc(~"100% - 20px");
-      }
```

Comment:
Is this no longer used on v1 search?

Reply:
I moved that to its own file, which I then forgot to commit in this PR.  I'm adding a commit with new files now.